app/queue/event_producer.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, config):
         self._kafka_producer = KafkaProducer(bootstrap_servers=config.bootstrap_servers, **config.kafka_producer)
         self.topics = {Topic.egress: config.host_egress_topic, Topic.events: config.event_topic}
 
-    def write_event(self, event, key, headers, topic):
+    def write_event(self, event, key, headers, topic, wait=False):
```

Comment:
Making this a keyword-only argument may make calls more readable.

```suggestion
    def write_event(self, event, key, headers, topic, *, wait=False):
```

Also thinking about alternative names. Like _wait_for_kafka_, _blocking_ or _synchronous_. _wait_ is fine though.